cli/cmd/volume.go

```diff
@@ -152,16 +154,44 @@ var volumeResetCmd = &cobra.Command{
 	},
 }
 
+func volumeInfoHandler2(cmd *cobra.Command)(api.Volinfo, error) {
+	var vols api.Volinfo
+	validateNArgs(cmd, 0, 1)
+	volname := ""
+	if len(cmd.Flags().Args()) > 0 {
+		volname = cmd.Flags().Args()[0]
+	}
+	if volname == ""{
+		vols_, err := client.VolumeAll()
```

Comment:
same function can be used here

client.Volumes("")

Reply:
Rectified!